### DISCUSSION TITLE
[Feature Request]: List of proxies

### DISCUSSION THREAD

**Discussion by benjam-es** [beyondcode/herd-community#377] [Ideas]
### Feature Description

It would be useful to have a tab similar to the site list for "proxies" so they can be viewed/secured etc

### Is this feature valuable for other users as well and why?

For anyone who uses proxies with Laravel Valet currently, this is a limitation with Herd - so this will be helpful to all users making use of the proxies capability within Valet.

**Comment by boobooking** [comment]
I also need this feature very much. Actually proxy functionality is the only reason I use Valet - all sites live in separate docker containers on different ports.

**Comment by karlatascent** [comment]
+1 for this feature
